scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ev
+testchange=`git diff --exit-code -- docs > /dev/null`
+if [$testchange]; then
+  deploy:
```

Comment:
All I can think of is this is not bash shell (the name and syntax suggests this should be a bash shell).

I will double check the Travis syntax tomorrow.

Btw, what is the rationale for this PR ? Is this a simple optimization (e.g. it does not fix anything) to save Travis resources ?

Reply:
@ggouaillardet I asked a question on the MTT devel list to see if we could not re-deploy every time there's a commit to MTT.  It causes a git push + email in the docs tree, and the only thing it contains is a timestamp update in the latex source.  E.g.: https://github.com/open-mpi/mtt/commit/41bbb3c184a1fa4e796bce338862a8ec39b7bd7f

Reply:
Yeah, I'm not convinced this is actually worth all this angst, but was willing to spend five minutes on it just to keep the git repo from unnecessarily growing. If it goes back/forth much more, though, I'll drop it as not worth the time.